custom_components/tuya_local/devices/djive_arc_fan.yaml

```diff
@@ -0,0 +1,101 @@
+name: djive Flowmate ARC Humidifier
```

Comment:
what about `Fan humidifier combo` or shall I still include a reference to the brand? like `djive Fan` or `ARC Fan`. Just asking because I looked at other config files, and sometimes the brand or model are included and sometimes not.

Reply:
Its more of a newer policy, so older configs still contain branding in the main name. But without branding is best. Just "Humidifying fan" is probably ok.